recipes/bioconductor-txdb.hsapiens.ucsc.hg38.knowngene/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: bioconductor-txdb.hsapiens.ucsc.hg38.knowngene
+  version: 3.4.0
+
+
+# Keeping the sources in here should trigger any warnings about missing URLs or
+# hash mismatches at build time, even though nothing is actually done with the
+# downloaded package.
+source:
+  fn: TxDb.Hsapiens.UCSC.hg38.knownGene_3.4.0.tar.gz
+  url:
+    - http://bioconductor.org/packages/3.5/data/annotation/src/contrib/TxDb.Hsapiens.UCSC.hg38.knownGene_3.4.0.tar.gz
+    - https://depot.galaxyproject.org/software/TxDb.Hsapiens.UCSC.hg38.knownGene/TxDb.Hsapiens.UCSC.hg38.knownGene_3.4.0_src_all.tar.gz
```

Comment:
Technically no, but keeping them in here checks to make sure they are available and checks the hash at build-time. This is at the expense of extra download time/space though. I could go either way on this. I wish there were a good way of pulling out the meta.yaml URLs in the post-link script, then the URLs and hash could be maintained, as normal, in the meta.yaml.